Geth 1.4 will not start on ARM 64 (AArch64)

#### System information

Geth version: 

(compiled from source)
Geth
Version: 1.4.0-rc
Git Commit: 8241fa522772837f0d05511f20caa6da1d5a3209
Protocol Versions: [63 62 61]
Network Id: 1
Go Version: go1.6.2
OS: linux
GOPATH=
GOROOT=/usr/lib/go

(downloaded binary)
Geth
Version: 1.4.0-rc
Git Commit: 8241fa522772837f0d05511f20caa6da1d5a3209
Protocol Versions: [63 62 61]
Network Id: 1
Go Version: go1.6.1
OS: linux
GOPATH=
GOROOT=/usr/local/go

OS & Version: Linux - Arch Linux Arm on Odroid C2 (AArch64)

#### Expected behaviour

geth starts successfully

#### Actual behaviour

geth does not start successfully

#### Steps to reproduce the behaviour

try to start geth from command line

#### Error

```
$ geth
I0424 16:12:07.718054 ethdb/database.go:82] Alloted 128MB cache and 1024 file handles to /home/xxx/.ethereum/chaindata
I0424 16:12:08.134728 ethdb/database.go:169] closed db:/home/xxx/.ethereum/chaindata
I0424 16:12:08.138212 ethdb/database.go:82] Alloted 128MB cache and 1024 file handles to /home/xxx/.ethereum/chaindata
panic: notify: epoll_wait(2) error function not implemented

goroutine 66 [running]:
panic(0xc1e2c0, 0x48205d8010)
	/usr/lib/go/src/runtime/panic.go:481 +0x384
github.com/rjeczalik/notify.(*inotify).loop(0x48200b0000, 0x4820092120)
	/home/xxx/build/geth-git/src/geth-git/build/_workspace/src/github.com/ethereum/go-ethereum/Godeps/_workspace/src/github.com/rjeczalik/notify/watcher_inotify.go:215 +0x568
created by github.com/rjeczalik/notify.(*inotify).lazyinit
	/home/xxx/build/geth-git/src/geth-git/build/_workspace/src/github.com/ethereum/go-ethereum/Godeps/_workspace/src/github.com/rjeczalik/notify/watcher_inotify.go:133 +0x210
```

Geth worked fine as of version 1.3.6. Upgrade to 1.4 introduced inability to start.

--- a/accounts/watch.go
+++ b/accounts/watch.go
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-// +build darwin,!ios freebsd linux netbsd solaris windows
+// +build darwin,!ios freebsd linux,!arm64 netbsd solaris windows
 
 package accounts
 

--- a/accounts/watch_fallback.go
+++ b/accounts/watch_fallback.go
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-// +build ios !darwin,!freebsd,!linux,!netbsd,!solaris,!windows
+// +build ios linux,arm64 !darwin,!freebsd,!linux,!netbsd,!solaris,!windows
 
 // This is the fallback implementation of directory watching.
 // It is used on unsupported platforms.

